unturned/OpenMod.Unturned/Environment/Events/UnturnedDayNightUpdatedEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenMod.Core.Eventing;
+
+namespace OpenMod.Unturned.Environment.Events
+{
+    public class UnturnedDayNightUpdatedEvent : Event
+    {
+        public bool IsDayTime { get; }
```

Comment:
This should be an enum instead of a bool (e.g. DayTime.Day & DayTime.Night)

Reply:
Resolved by 290229f